source/security-guide/cmmc-compliance.rst

```diff
@@ -0,0 +1,90 @@
+
+**User Agreement and Access Approval**: Mattermost Enterprise supports a :doc:`Custom Terms of Service </administration-guide/comply/custom-terms-of-service>` banner that users must accept upon first login. This can be used to remind users of acceptable use policies or consent to monitoring, indirectly supporting training/awareness requirements and ensuring users acknowledge security terms before accessing CUI.
+
+Authentication and Multi-Factor Authentication (MFA)
+-----------------------------------------------------
+
+Secure authentication is critical for protecting Controlled Unclassified Information (CUI). Mattermost offers several features to strengthen user authentication in alignment with CMMC requirements:
+
+**Unique User Identification**: Each Mattermost user has a unique account (username/email), satisfying the need for unique IDs (IA 3.5.1). Administrators can disable generic or shared accounts and require login via enterprise SSO or individual credentials, ensuring accountability.
+
+**Password Policy Enforcement**: For built-in authentication, Mattermost administrators can :ref:`enforce strong password requirements <administration-guide/configure/authentication-configuration-settings:password>` (minimum length, complexity). This helps meet IA 3.5.2 by requiring robust passwords and reducing the risk of credential compromise.
+
+:doc:`Multi-Factor Authentication </administration-guide/onboard/multi-factor-authentication>`: Mattermost supports MFA for all user accounts. In self-hosted deployments, admins can enable and enforce TOTP-based MFA (e.g. requiring a one-time code from Google Authenticator during login). When Mattermost is integrated with SSO (SAML/OIDC), you can leverage the IdP’s MFA policies (e.g. CAC/PIV or OTP) for Mattermost logins. Requiring two factors for authentication aligns with CMMC practice IA 3.5.3, adding an extra layer of verification to protect accounts even if passwords are compromised.
+
+**Account Lockout and Recovery**: Mattermost can limit failed login attempts and lock accounts after a specified number of failures, helping to thwart brute-force attacks (IA 3.5.3, additional aspect). It also provides options for :ref:`secure password reset <end-user-guide/collaborate/access-your-workspace:reset your password>` or :ref:`administrator-issued password resets <administration-guide/configure/user-management-configuration-settings:reset user's password>` to support account recovery while maintaining security controls.
+
+Audit Logging and Accountability
+---------------------------------
+
+CMMC Level 2 (`NIST 800-171 <https://csrc.nist.gov/pubs/sp/800/171/r2/upd1/final>`_) places heavy emphasis on audit logging and the ability to track and monitor system activity (Audit & Accountability, AU 3.3.x controls). Mattermost provides built-in logging and monitoring features that help meet these requirements:
```

Comment:
Curious about the hyperlink here to NIST? None of the other section intros have one. Is this specific to Audit Logging? If not, suggest we remove.